packages/web/src/components/tipping/feed-tip-tile/FeedTipTile.tsx

```diff
@@ -188,19 +188,8 @@ const MIN_USERS_TO_BUTTONS_MARGIN = 8
 export const FeedTipTile = () => {
   const isTippingEnabled = getFeatureEnabled(FeatureFlags.TIPPING_ENABLED)
 
-  const [containerPosition, setContainerPosition] = useState<Nullable<DOMRect>>(
-    null
-  )
-  const containerCallbackRef = useCallback((node: HTMLDivElement) => {
-    setContainerPosition(node.getBoundingClientRect())
-  }, [])
-
-  const [buttonsPosition, setButtonsPosition] = useState<Nullable<DOMRect>>(
-    null
-  )
-  const buttonsCallbackRef = useCallback((node: HTMLDivElement) => {
-    setButtonsPosition(node.getBoundingClientRect())
-  }, [])
+  const { position: containerPosition, ref: containerRef } = usePosition()
```

Comment:
doubt: can we call the same hook, as I'm doing here, with no arguments in the hook, and return diff objects? when i tested on the client, it works, meaning I did not get the same object twice. but wondering if it's safe... could we possibly get the same object twice here in these 2 `usePosition()` hook calls?